lib/db_pool/core/importer.ex

```diff
@@ -54,7 +54,12 @@ defmodule DbPool.Core.Importer do
   end
 
   defp import_postgres_db(%Database{} = database, filename, dir) do
-    {_, 0} = System.cmd("createdb", [database.name])
+    case System.cmd("createdb", [database.name]) do
+      {_, 0} ->
+        Logger.info("The database has been created")
+      {_, 1} ->
```

Comment:
We wanted to fail here 😐

Reply:
How would we go about re-importing PostgreSQL then?
If there is an existing database, we obviously can't create one with that name. So I thought of dropping it, create and import.

But this is not required as per mysql adapter case, where we already handle existing database situation, which is what @iffyuva said.

Reply:
So as of [this](https://github.com/dockup/db-pool/pull/20/files#r307237165) , we should probably not fail the whole process (which would shutdown the app), but if we have a better way to capture what actually happened, then it can be helpful 😄